.npm/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
A Note: Copy license file as build step.
Now we would have two wrong copyright years :)

Reply:
Shure! The same goes for the `version` which should later be the same as the current release version. 

(Symlinks don't work with `npm publish` unfortunately.)